journal/journal-page.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./journal-style.css">
+    <script src="./journal-app.js"></script> <!--Should we use defer keyword?-->
```

Comment:
please raise this up for discussion to the team so others can help contribute to decision